### ISSUE TITLE
Typescript error

### ISSUE THREAD

**Issue by kokororin** [valeriangalliat/fetch-cookie#70]
Code:
```ts
import nodeFetch from 'node-fetch';
const fetch = fetchCookie(nodeFetch);
```

```
error TS2345: Argument of type 'typeof fetch' is not assignable to parameter of type 'GenericFetch<GenericRequestInfo, RequestInit, Response>'.
  Types of parameters 'url' and 'input' are incompatible.
    Type 'GenericRequestInfo' is not assignable to type 'RequestInfo'.
      Type 'GenericRequest' is not assignable to type 'RequestInfo'.
        Type 'GenericRequest' is missing the following properties from type 'Request': clone, context, headers, method, and 17 more.


```

versions in package.json
```
    "@types/node-fetch": "^2.6.1",
    "node-fetch": "^2.6.1",
```

**Comment by valeriangalliat** [comment]
Interesting. This version of node-fetch declares `RequestInfo` as `string | URLLike | Request`, with `URLLike` being ` { href: string }`.

I don't believe that's part of the fetch standard which is why TypeScript complains that it's incompatible, even though it doesn't actually cause any problem...

The error message is also really misleading, but removing `URLLike` did allow TypeScript to compile so that's indeed the real issue. :sweat_smile: 

To be honest I think your best solution is to do `const fetch = fetchCookie(nodeFetch as any)`

**Comment by valeriangalliat** [comment]
Alright I just published fetch-cookie version 2.0.3 which should fix this issue, let me know if that works for you!

**Comment by kokororin** [comment]
Thanks for the fast fix! It works now.
